<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="CreateButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>330</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Cart</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="petTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>161</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QTableWidget" name="bundle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>461</width>
      <height>121</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <row/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="PushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Database</string>
    </property>
   </widget>
   <widget class="AddButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to Cart</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="holder">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>256</width>
      <height>101</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <row/>
    <column/>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <zorder>holder</zorder>
   <zorder>bundle</zorder>
   <zorder>petTable</zorder>
   <zorder>CreateButton</zorder>
   <zorder>pushButton</zorder>
   <zorder>addButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header location="global">pushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AddButton</class>
   <extends>QPushButton</extends>
   <header location="global">addbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
